lib/6to5/transformation/templates/umd-runner-body.js

```diff
@@ -2,6 +2,6 @@
   if (typeof define === "function" && define.amd) {
     define(AMD_ARGUMENTS, factory);
   } else if (typeof exports !== "undefined") {
```

Comment:
It seams risky to just assume `module` to exist just because `exports` does. Even if it is the case 99.9% of the time. I can imagine fake environments, like [browserify][1], that might only define `exports`. Better safe the sorry I guess.

[1]: //substack/node-browserify